benchmarks/common.py

```diff
@@ -285,6 +284,10 @@ def print_summary(filename):
                 print(col.ljust(width), f"{data[col].mean():.1%}")
             elif col in ("graphs", "graph_calls", "captured_ops", "total_ops"):
                 print(col.ljust(width), f"{data[col].mean():.1f}")
+            elif col in ("compilation_latency"):
+                print(col.ljust(width), f"mean={data[col].mean():.1f} seconds")
+            elif col in ("compression_ratio"):
```

Comment:
I find the naming is a little confusing here. First, having `memory` in the naming can make the meaning more explicit. Second, I guess the reason you picked compression_ratio `eager_peak_mem / dynamo_peak_mem`(instead of the inverse) to measure the memory reduction effect is probably following how we compute the speedup, but I feel for memory reduction the metric is uncommon? But that could be a personal preference.

Reply:
I have typically used memory compression ratio in the research before. Higher the compression ratio, better the technique. It kind of makes graphs/conclusions easy to read. 

Another popular one is percentage reduction in memory. I think it's compression ratio is ok for now. I can add `memory` to be more precise.